### DISCUSSION TITLE
✔ MVM Machine Attacks server without any additional money or regen

### DISCUSSION THREAD

**Discussion by leeroytm** [TitanTF/TitanTF#39] [General]
Reasons: the main reason why i dont play on the servers anymore is cuz of the additional money
Gamemode: Machine Attack

Suggestion by: Squid#2266 <@386788881720410126>
Located: https://discord.com/channels/774245035615977473/845235964712583208/852369125572345877

---
### Discssion happened:
> Majin Devil — 06/10/2021
Won't make much difference, most ma maps are easy to farm up to max

> Green — 06/10/2021
they just idle in ready up state and wait til max money
then goes super saiyan mode

> Apples ( ﾟ∀ ｡) — 06/10/2021
toggle the money regen via vote

**Comment by jiaweing** [comment]
This one is resolved with the new money regen toggle vote.
